src/pages/audioPAge/audioPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './style.scss';
+import audioBook from './audioBook.mp3';
+import AudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+
+const AudioPage = () => {
+  const bookName = 'Mind without body';
+  const audio = audioBook;
+  const BookCover = 'https://via.placeholder.com/353x498';
+  const description = `is simply dummy text of the printing and typesetting industry.
+  Lorem Ipsum has been the industry's standard dummy text ever since
+  the 1500s, when an unknown printer took a galley of type and
+  scrambled it to make a type specimen book.galley of type and
+  scrambled it to make a type specimen book. .galley of type and
+  scrambled it to make a type specimen book.type specimen book`;
```

Comment:
In the your other PR you made those as an object
So ether change those to object or change those back to a regular variables

mu suggestion here is like this 
```suggestion
 const book = { 
      bookName = 'Mind without body';
      audio = audioBook;
      bookCover = 'https://via.placeholder.com/353x498';
      description = `is simply dummy text of the printing and typesetting industry.
  Lorem Ipsum has been the industry's standard dummy text ever since
  the 1500s, when an unknown printer took a galley of type and
  scrambled it to make a type specimen book.galley of type and
  scrambled it to make a type specimen book. .galley of type and
  scrambled it to make a type specimen book.type specimen book`;
  
  const {bookName, audio, bookCover, description } = book
  
```